onboarding/github.md

```diff
@@ -0,0 +1,23 @@
+# GitHub
+
+#### Organisation
+
+NeuroPoly has a github organisation:
+https://github.com/neuropoly
```

Comment:
Actually there are 7 others too:

https://github.com/neuropoly/onboarding/blob/088cce204505381b3f41a2d10eec5447111f255e/.github/ISSUE_TEMPLATE/membership-card.md?plain=1#L151

I've been hesitant to document this here because it's hard to tell what should go where. This information overlaps with what's on https://neuro.polymtl.ca/software.html and should probably take https://intranet.neuro.polymtl.ca/agenda-and-calendar.html into account somehow too.